syft/generic/object_storage.py

```diff
@@ -2,11 +2,11 @@
 from typing import List
 from typing import Union
 
+from syft.exceptions import ObjectNotFoundError
 from syft.generic.frameworks.types import FrameworkTensor
 from syft.generic.frameworks.types import FrameworkTensorType
 from syft.generic.tensor import AbstractTensor
-
-from syft.exceptions import ObjectNotFoundError
+from syft.workers.abstract import AbstractWorker
 
 
 class ObjectStorage:
```

Comment:
Is it an `ObjectStorage` or an `ObjectStore`?

Reply:
Yeah, I should have renamed this. Will update.